modules/osd-applications-config-custom-domains.adoc

```diff
@@ -0,0 +1,102 @@
+// Module included in the following assemblies for OSD and ROSA:
+//
+// * applications/deployments/osd-config-custom-domains-applications.adoc
+
+[id="osd-applications-config-custom-domains.adoc_{context}"]
+= Configuring custom domains for applications
+
+Custom domains are specific wildcard domains that can be used with {product-title} applications. The top-level domains (TLDs) are owned by the customer that is operating the {product-title} cluster. The Custom Domains Operator sets up a new `ingresscontroller` with a custom certificate as a second day operation. The public DNS record for this `ingresscontroller` can then be used by an external DNS to create a wildcard CNAME record for use with a custom domain.
+
+.Prerequisites
+
+* A user account with `dedicated-admin` privileges
+* A unique wildcard domain, such as ``*.apps.<company_name>.io`
+* A wildcard custom certificate, such as `CN=*.apps.<company_name>.io`
+* Access to a cluster with the latest version of the `oc` CLI installed
+
+[IMPORTANT]
+Do not use the reserved names `default` or `apps*`, such as `apps` or `apps2`, in the `metadata/name:` section of the `CustomDomain` CR.
+
+.Procedure
+
+. Create a new TLS secret from a private key and a public certificate, where `fullchain.pem` and `privkey.pem` are your public or private wildcard certificates.
++
+.Example
++
+
+[source,terminal]
+----
+$ oc create secret tls <name>-tls --cert=fullchain.pem --key=privkey.pem -n <my_project>
+----
+
+. Create a new `CustomDomain` custom resource (CR). For example, using the file `<company_name>-custom-domain.yaml`, `<name>-tls` is the secret created in the previous step, and `apps.companyname.io` is the custom domain.
```

Comment:
Could this be chopped down by moving the example YAML title into the example title?

You could also use callouts in the example to eliminate the second sentence.